protos/core.proto

```diff
@@ -424,4 +427,13 @@ message TableMetadata {
   Target target = 1;
   string type = 2; // "table" or "view"
   repeated Field fields = 3;
+  int64 last_updated_millis = 4;
+}
+
+message PersistedTableMetadata {
+  Target target = 1;
+  string definition_hash = 2; // sha1 hash of ExectionAction object
+  int64 last_updated_millis = 3; // when the table/view last updated, extracted from warehouse
```

Comment:
this comment seems a bit confusing to me. 'extracted from warehouse' makes it sound like it's the warehouse's last modified timestamp, which it is, but it sounds like this value might change even if no runs happen (which it can't, right?).

i would suggest `Timestamp of the last modification to this dataset (where that modification happened as a result of a Dataform run).`